src/components/RegisterAccount2.tsx

```diff
@@ -83,77 +80,65 @@ export const RegisterAccount2 = (props: RegisterAccount2Props) => {
                   <FormControl
                     onChange={e => setFirstName(e.target.value)} 
                     value = {firstName}/>
-              </FormGroup>
-            </Col>
-            <Col xs={12} sm={6}>
-              <FormGroup>
+                </FormGroup>
+                <FormGroup>
                   <FormLabel>Last Name</FormLabel>
                   <FormControl
                     onChange={e => setLastName(e.target.value)} 
                     value = {lastName}/>
-              </FormGroup>
-            </Col>
-            <Col xs={12} sm={6}>
-              <FormGroup className='required'>
+                </FormGroup>
+                <FormGroup className='required'>
                   <FormLabel>Password</FormLabel>
                   <FormControl
                     type="password"
                     onChange={e => setPassword1(e.target.value)} 
                     value = {password1}/>
-              </FormGroup>
-            </Col>
-            <Col xs={12} sm={6}>
-              <FormGroup className='required'>
-                  <FormLabel>Confirm password</FormLabel>
-                  <FormControl
-                    type="password"
-                    onChange={e => setPassword2(e.target.value)} 
-                    value = {password2}/>
-              </FormGroup>
-            </Col>
-          </Row>
-          <div className="buttonsContainer">
-            <Button
-              variant='primary'
-              onClick={onCreateAccount}
-              type="button"
-              style={{ marginLeft: 20 }}
-              disabled={isLoading}
-            >
-              Continue <IconSvg icon='arrowForward' />
-            </Button>
+                </FormGroup>
+                <FormGroup className='required'>
+                    <FormLabel>Confirm password</FormLabel>
+                    <FormControl
+                      type="password"
+                      onChange={e => setPassword2(e.target.value)} 
+                      value = {password2}/>
+                </FormGroup>
+                <Button
+                  variant='contained'
+                  onClick={onCreateAccount}
+                  type="button"
+                  disabled={isLoading}
+                >
+                  Continue
+                </Button>
+              </div>
+          </div>
+          <div className={'panel-right'}>
+            <div className={'panel-right-text'}>
+              <h3>Email address verified!</h3>
+              <h4>Now complete your registration.</h4>
+              <p>Your <strong>username</strong> can be up to 20 letters and numbers, but no spaces.</p>
+              <p><strong>First and last names</strong> are optional, but recommended because they make it easier for team members to find you.</p>
+              <p>We recommend using a strong, unique <strong>password</strong> of between 16-32 characters. You can use letters, numbers, and punctuation marks.</p>
```

Comment:
I don't think these are the correct constraints (I think they are conditions that Adam made up for the design).  Could you try to change your password to something invalid?  The backend error should elucidate the constraints.
Similarly with the username.